core/templates/dev/head/css/oppia.css

```diff
@@ -771,7 +772,7 @@ textarea {
 }
 @media (max-width: 440px) {
   .oppia-footer-padding-below-banner{
```

Comment:
I see that you didn't do this, but please put one space between the CSS selector and the opening bracket